Sources/StreamChatUI/Appearance+SwiftUI.swift

```diff
@@ -5,6 +5,7 @@
 import Combine
 import StreamChat
 import SwiftUI
+import SwiftUICore
```

Comment:
What error were you seeing that this import fixed? (i'm just curious)

Reply:
```bash
Error: cannot find type 'SwiftUICore' in scope
```

The failed job is in the PR description, easier to look at all the error messages at once.